symbolic-base/src/ZkFold/Symbolic/Compiler/ArithmeticCircuit/Internal.hs

```diff
@@ -211,43 +172,55 @@ instance
     type WitnessField (ArithmeticCircuit a p i) = WitnessF a (WitVar p i)
     witnessF (behead -> (c, o)) = o <&> \case
```

Comment:
With optimization you introduced, it is possible to just use `at` here instead of pattern-matching

Reply:
I'm talking about just `at <$> o`, so `\case` is not needed here at all